vm/vm_linux_test.go

```diff
@@ -103,9 +103,68 @@ func TestCPUAMD64(t *testing.T) {
 		}
 		t.Logf("io %s = %q", tt.args, string(b))
 	}
+}
+
+// TestCPUARM tests both general and specific things. The specific parts are the io and cmos commands.
+// It being cheaper to use a single generated initramfs, we use the full u-root for several tests.
+func TestCPUARM(t *testing.T) {
+	d := t.TempDir()
```

Comment:
future work - would it be convenient to wrap all of the tempdir, vm.New, i.Uroot, start VM, etc. all in a function?